sections/semantics-document-metadata.include

```diff
@@ -355,15 +355,16 @@
     <dd>
       <pre class="idl" data-highlight="webidl">
         interface HTMLLinkElement : HTMLElement {
-          attribute DOMString href;
-          attribute DOMString? crossOrigin;
-          attribute DOMString rel;
-          attribute DOMString rev;
-          [SameObject, PutForwards=value]readonly attribute DOMTokenList relList;
-          attribute DOMString media;
-          attribute DOMString hreflang;
-          attribute DOMString type;
-          [SameObject, PutForwards=value] readonly attribute DOMTokenList sizes;
+          [CEReactions] attribute USVString href;
```

Comment:
or define it... also should be seperate issue. I didn't look very carefully for as definition so maybe we do - can you please follow that up?